assets/airports/lrop.json

```diff
@@ -0,0 +1,1409 @@
+                ["N44d13m24.354", "E26d01m22.238"],
+                ["N44d13m06.407", "E26d00m49.911"],
+                ["N44d12m50.412", "E26d00m15.663"],
+                ["N44d12m36.475", "E25d59m39.721"],
+                ["N44d12m24.688", "E25d59m02.324"],
+                ["N44d12m15.129", "E25d58m23.719"],
+                ["N44d12m07.862", "E25d57m44.162"],
+                ["N44d12m02.934", "E25d57m03.915"],
+                ["N44d12m00.379", "E25d56m23.245"],
+                ["N44d12m00.212", "E25d55m42.422"],
+                ["N44d12m02.437", "E25d55m01.714"],
+                ["N44d12m07.036", "E25d54m21.393"],
+                ["N44d12m13.981", "E25d53m41.724"],
+                ["N44d12m23.225", "E25d53m02.972"],
+                ["N44d12m34.707", "E25d52m25.391"],
+                ["N44d12m48.351", "E25d51m49.233"],
+                ["N44d13m04.066", "E25d51m14.736"],
+                ["N44d13m21.748", "E25d50m42.129"],
+                ["N44d13m41.281", "E25d50m11.630"],
+                ["N44d14m02.534", "E25d49m43.440"],
+                ["N44d14m25.367", "E25d49m17.748"],
+                ["N44d14m49.628", "E25d48m54.725"],
+                ["N44d15m15.156", "E25d48m34.524"],
+                ["N44d15m41.782", "E25d48m17.281"],
+                ["N44d16m09.329", "E25d48m03.111"],
+                ["N44d16m37.614", "E25d47m52.110"],
+                ["N44d17m06.449", "E25d47m44.352"],
+                ["N44d17m35.641", "E25d47m39.890"],
+                ["N44d18m04.998", "E25d47m38.755"],
+                ["N44d18m34.324", "E25d47m40.956"],
+                ["N44d19m03.423", "E25d47m46.479"],
+                ["N44d19m32.102", "E25d47m55.290"],
+                ["N44d20m00.170", "E25d48m07.331"],
+                ["N44d20m27.439", "E25d48m22.522"],
+                ["N44d20m53.728", "E25d48m40.764"],
+                ["N44d21m18.861", "E25d49m01.937"],
+                ["N44d21m42.672", "E25d49m25.899"],
+                ["N44d22m05.000", "E25d49m52.492"],
+                ["N44d22m25.697", "E25d50m21.540"],
+                ["N44d22m44.624", "E25d50m52.848"],
+                ["N44d23m01.655", "E25d51m26.209"],
+                ["N44d23m16.677", "E25d52m01.399"],
+                ["N44d23m29.589", "E25d52m38.184"],
+                ["N44d23m40.304", "E25d53m16.319"],
+                ["N44d23m48.751", "E25d53m55.548"],
+                ["N44d23m54.874", "E25d54m35.610"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_FAF08L": ["N44d33m53.000", "E25d55m43.300"],
+        "_FAF08R": ["N44d33m05.200", "E25d54m15.600"],
+        "_FAF26L": ["N44d34m51.400", "E26d17m38.200"],
+        "_FAF26R": ["N44d35m29.800", "E26d17m02.600"],
+        "_THR08L": ["N44d34m35.840", "E26d05m02.760"],
+        "_THR08R": ["N44d33m52.780", "E26d04m36.290"],
+        "_THR26L": ["N44d34m04.690", "E26d07m13.540"],
+        "_THR26R": ["N44d34m47.730", "E26d07m39.950"],
+        "ABRUT"  : ["N44d27m00.921", "E25d10m59.401"],
```

Comment:
They don't need to be aligned with the invisible fixes so you can get rid of 2 spaces for all fixes